dev/tasks/java-jars/github.yml

```diff
@@ -164,6 +188,18 @@ jobs:
           arrow/ci/scripts/java_full_build.sh \
             $GITHUB_WORKSPACE/arrow \
             $GITHUB_WORKSPACE/arrow/java-dist
+          arrow/ci/scripts/java_full_build.sh \
+            $GITHUB_WORKSPACE/arrow \
+            $GITHUB_WORKSPACE/arrow/java-dist/x86_64
+          arrow/ci/scripts/java_full_build.sh \
+            $GITHUB_WORKSPACE/arrow \
+            $GITHUB_WORKSPACE/arrow/java-dist/aarch_64
```

Comment:
Why do we need this?
I think that we need to bundle all `.so`s/`.dylib`s/.`dll`s to one `.jar`.
It seems that we don't need to do them because our pattern matches all `java-dist/**/*arrow_cdata_jni.*`: https://github.com/apache/arrow/blob/master/java/c/pom.xml#L70-L79

Reply:
Removed. Let's see what's the outcome.